extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/HudiCatalogRangerSparkExtensionSuite.scala

```diff
@@ -206,4 +207,56 @@ class HudiCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
                |""".stripMargin)))(s"does not have [select] privilege on [$namespace1/$table1]")
     }
   }
+
+  test("IndexBasedCommand") {
+    assume(!isSparkV33OrGreater, "Create index have compatible for spark 3.3 and greater version")
```

Comment:
Have told my teammate to fix this issue in hudi, but here we need to ignore these version's test..

Reply:
for such case, could you please leave a JIRA ticket(once it is created) here

Reply:
> for such case, could you please leave a JIRA ticket(once it is created) here

Yea, will change after create the ticket

Reply:
Fine to me for skipping this test on Spark 3.3 or greater. But the clue message shows the opposite meaning. Or just "Hudi index creation not supported on Spark 3.3 or greater currently"